test/reader_test.cpp

```diff
@@ -156,7 +157,14 @@ TEST_CASE("GetAllChildren Test", "[Reader] ")
     }
 }
 
-TEST_CASE("Point Error Handling Test", "[Reader] ")
+// TODO[Leo]: Make this test optional
+TEST_CASE("GetAllPoints Test", "[Reader]")
+{
+    //    FileReader reader("test/data/autzen-classified.copc.laz");
+    //    REQUIRE(reader.GetAllPoints().Get().size() == reader.GetLasHeader().point_count);
```

Comment:
we should make a smaller test file for these kinds of tests

Reply:
Agreed